lldb/source/Host/windows/ProcessLauncherWindows.cpp

```diff
@@ -21,42 +21,41 @@
 using namespace lldb;
 using namespace lldb_private;
 
-static void CreateEnvironmentBuffer(const Environment &env,
-                                    std::vector<char> &buffer) {
-  // The buffer is a list of null-terminated UTF-16 strings, followed by an
-  // extra L'\0' (two bytes of 0).  An empty environment must have one
-  // empty string, followed by an extra L'\0'.
+std::vector<wchar_t>
+ProcessLauncherWindows::CreateEnvironmentBufferW(const Environment &env) {
```

Comment:
I don't understand why this cannot remain an internal detail of the implementation and needs to be moved into `ProcessLauncherWindows`. The equivalent signature would be:

```c++
std::vector<wchar_t> CreateEnvironmentBufferW(const Environment &);
```